docs/weaviate/concepts/data-import.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Data import
+sidebar_position: 11
+description: "Theoretical explanation of client-side and server-side batch imports."
+image: og/docs/concepts.jpg
+---
+
+Weaviate offers two flexible methods for importing data in bulk: **client-side batching** and **server-side batching**. This allows you to choose the best strategy based on your specific needs.
+
+- **Client-side batching** <br/>
+  In the traditional client-side approach, **your application is responsible for grouping data into batches**. You define the exact size of each batch (e.g., 100 objects) using the appropriate [client library method](../manage-objects/import.mdx). The client then sends these pre-defined chunks to the Weaviate server.
+
+  This method gives you direct control over the import process but may require manual tuning of parameters like `batch_size` and `concurrent_requests` to achieve optimal performance and avoid overwhelming the server.
```

Comment:
This method gives you direct control over the import process  through manual tuning of parameters like `batch_size` and `concurrent_requests`. However, the tuning must be done "blindly" on the client side, without knowledge of the server status.